### PR TITLE
ER351 & ER403 QA Referral flow

### PR DESCRIPTION
**Pull request by leewadhams**

### CODE REVIEW

File: src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs [hunk 1]
```diff
@@ -34,11 +34,22 @@ private static DashboardViewModel MapToViewModel(IEnumerable<VacancyReview> revi
...
         }
+
+        private static string CalculateStatus(VacancyReview review)
+        {
+            if (review.Status == ReviewStatus.UnderReview && review.ManualOutcome == ManualQaOutcome.Referred)
+            {
+                return review.Status.GetDisplayName();
+            }
+            
+            return "Submitted";
```
Comment (sachpatel) [inline src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs]: `return VacancyStatus.Submitted.GetDisplayName();`

File: src/QA/QA.Web/Controllers/ReviewController.cs [hunk 2]
```diff
@@ -26,11 +27,27 @@ public ReviewController(ReviewOrchestrator orchestrator)
...
 
             return RedirectToRoute(RouteNames.Dashboard_Index_Get);
         }
+
+        [HttpGet("referral", Name = RouteNames.Vacancy_Review_Referral_Get)]
+        public async Task<IActionResult> Referral([FromRoute] Guid reviewId) 
+        {
+            var vm = await _orchestrator.GetReferralViewModelAsync(reviewId);
+
+            return View("Review", vm);
```
Comment (sachpatel) [inline src/QA/QA.Web/Controllers/ReviewController.cs]: `ViewNames.Review`

File: src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs (hunk 1 above)
Comment (leewadhams) [reply src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs]: 👍

File: src/QA/QA.Web/Controllers/ReviewController.cs (hunk 2 above)
Comment (leewadhams) [reply src/QA/QA.Web/Controllers/ReviewController.cs]: 👍

**Comment by leewadhams** [comment]
Good spot on the ReviewedByUser! Was merged out.

**Pull request merged**